### PR TITLE
Reformatted "DP More" Module

### PR DESCRIPTION
**Pull request by dustin-miao**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: content/6_Advanced/DP_More.mdx [hunk 1]
```diff
@@ -54,7 +70,13 @@ Queue with 2 stacks is used to remove a factor of $\mathcal{O}(\log N)$ in [USAC
...
 </Optional> -->
 
-### "Connected Component" DP Problems
+### Connected-Component DP Problems
+
+<IncompleteSection>
+
+Make Permutations II a focus problem, move analysis to module.
+
+</IncompleteSection>
```
Comment (SansPapyrus683) [inline content/6_Advanced/DP_More.mdx:79]: do this yourself for this to be counted as an actual contribution

Comment (dustin-miao) [reply content/6_Advanced/DP_More.mdx:79]: what

Comment (dustin-miao) [reply content/6_Advanced/DP_More.mdx:79]: so the work I've done here isn't a contribution?

Comment (dustin-miao) [reply content/6_Advanced/DP_More.mdx:79]: I'm not getting paid to do this nor do I have any obligation to do so, so any work is a contribution?

Comment (SansPapyrus683) [reply content/6_Advanced/DP_More.mdx:79]: anyways I corrected myself earlier, so let’s just stop this

**Review by nwatx — APPROVED** [review]

**Pull request merged**